test/regress/regress1/proofs/macro-res-exp-singleton-after-elimCrowd.smt2

```diff
@@ -0,0 +1,384 @@
+; COMMAND-LINE: --proof-new --check-proofs-new
```

Comment:
Could we just enable this benchmark without specifically asking for proofs? (We will be testing proofs on all regressions eventually anyways).

The same applies to the regression you added on the earlier PR.

Reply:
Indeed! I'll fix.